.github/workflows/sync_readme.yml

```diff
@@ -0,0 +1,37 @@
+name: Sync README
```

Comment:
Using `/docs` directory in the main branch is better since we can remove the syncing?
https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site

scalardl javadocs are published with the way.
https://scalar-labs.github.io/scalardl/javadoc/ledger/

Reply:
Fixed to use `/root` of main branch on github pages in 70a8221.

We don't need `/docs` folder, `index.yaml`will updated under the root directory of the repo.